week1/prep-exercises/1-counter/src/components/Button.jsx

```diff
@@ -0,0 +1,11 @@
+
+function Button(props) {
+    const onClick = props.setFunc;
+    const buttonName = props.name;
+    const step = props.step;
```

Comment:
it's a lot better to use destructure object as you did in the `Count` component.